modules/47_regipol/regiCarbonPrice/postsolve.gms

```diff
@@ -221,92 +221,92 @@ display pm_taxCO2eq;
 
```

Comment:
Well, thx for adding but ye, puh, well, this is not great timing as I was doing a lot of changes here in a separate branch trying to do budget and year target at the same time. I guess, it will be very hard to merge that. :(

I will see how I make progress there and let's keep in touch on that.

Reply:
The changes I added shouldn't change much the structure of the code. You can probably compare the before and after my push, and you will be able to reproduce my changes to whatever version you have at your end.

If you have issues with that just share your work so far and I will try to help if I can.